src/DataTable/tests/ActionDisplay.test.jsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { mount } from 'enzyme';
+
+import ActionDisplay from '../ActionDisplay';
+import DataTableContext from '../DataTableContext';
+import BulkActions from '../BulkActions';
+import TableActions from '../TableActions';
+
+const instance = {
+  selectedFlatRows: null,
+  bulkActions: [],
+  tableActions: [],
+};
+
+const firstAction = {
+  buttonText: 'Primary action',
+  handleClick: () => {},
+  className: 'class1',
+};
+
+const secondAction = {
+  buttonText: 'Secondary action',
+  handleClick: () => {},
+  className: 'class2',
+};
+
+// eslint-disable-next-line react/prop-types
+const ActionDisplayWrapper = ({ value = instance, props = {} }) => (
+  <DataTableContext.Provider value={value}><ActionDisplay {...props} /></DataTableContext.Provider>
+);
+
+describe('<ActionDisplay />', () => {
+  it('renders null if there are no actions', () => {
+    const wrapper = mount(<ActionDisplayWrapper />);
+    expect(wrapper.find(ActionDisplay).text()).toEqual('');
+  });
+  it('displays bulk actions when rows are selected', () => {
+    const wrapper = mount(
+      <ActionDisplayWrapper
+        value={{ ...instance, bulkActions: [firstAction, secondAction], selectedFlatRows: [{}, {}] }}
+      />,
+    );
+    expect(wrapper.find(BulkActions)).toHaveLength(1);
+  });
+  it('does not display bulk actions when no rows are selected (no table actions)', () => {
+    const wrapper = mount(
+      <ActionDisplayWrapper
+        value={{ ...instance, bulkActions: [firstAction, secondAction], selectedFlatRows: [] }}
+      />,
+    );
+    expect(wrapper.find(ActionDisplay).text()).toEqual('');
+  });
+  it('displays tableActions', () => {
+    const wrapper = mount(
+      <ActionDisplayWrapper
+        value={{ ...instance, tableActions: [firstAction, secondAction], selectedFlatRows: [{}, {}] }}
+      />,
+    );
+    expect(wrapper.find(BulkActions)).toHaveLength(1);
```

Comment:
Is this supposed to be asserting for the existence of `BulkActions` when the test case is for displaying `tableActions`? 

Kind of curious why this is currently passing when, as far as I can tell, `ActionDisplayWrapper` should not be rendering `BulkActions`?

Reply:
OH THAT IS TRUE AND I DO NOT LIKE IT. Will investigate.